src/utils/build.js

```diff
@@ -309,6 +309,20 @@ const build = (zipPath, wdir) => {
       printStarting('Installing project dependencies');
       return runCommand('npm', ['install', '--production'], { cwd: tmpDir });
     })
+    .then(output => {
+      // `npm install` may fail silently without returning a non-zero exit code, need to check further here
+      const corePath = path.join(
+        tmpDir,
+        'node_modules',
+        constants.PLATFORM_PACKAGE
+      );
+      if (!fs.exists(corePath)) {
```

Comment:
`fs.exists` is both asynchronous and deprecated ([docs](https://nodejs.org/docs/latest-v6.x/api/fs.html#fs_fs_exists_path_callback)). I believe this needs to be `fs.existsSync` or one of the other methods mentioned in that doc link

Reply:
Nice catch! Fixed in 5fb57c1.